src/wizard/javascript/replay-onboarding/javascript/2.configure.md

```diff
@@ -0,0 +1,28 @@
+---
+name: JavaScript
+doc_link: https://docs.sentry.io/platforms/javascript/replay/
+support_level: production
+type: language
+---
+
+#### Configure
+
+Add the following to your SDK config. There are several privacy and performance options available, all of which can be set via the `integrations` constructor. Learn more about configuring Replay in our Github [Readme](https://github.com/getsentry/sentry-replay/blob/main/README.md).
+
+```javascript
+import * as Sentry from "@sentry/browser";
+import { Replay } from "@sentry/replay";
+
+Sentry.init({
+  dsn: "”https://examplePublicKey@o0.ingest.sentry.io/0",
+  integrations: [
+    new Replay({
+      // Capture 10% of all sessions
+      sessionSampleRate: 0.1,
+
+      // Of the remaining 90% of sessions, if an error happens start capturing
+      errorSampleRate: 1.0,
```

Comment:
These are the new config fields that we're offering, as of https://github.com/getsentry/sentry-replay/pull/268